api/R/add_database.R

```diff
@@ -0,0 +1,42 @@
+#' Add PEcAn database information to settings list
```

Comment:
Ok, as I said originally, I did think it was clear that you wanted to make this independent. Will leave as is, but we should take a closer look at `settings` and other packages to see if there are places they could/should use this new package.

That said, if you did use `pecanapi` in the main pecan code, that would nix the idea of making it a separate repo (which otherwise would be a fair thing to do, provided we don't anticipate this package to be highly sensitive to PEcAn versioning)

Reply:
Once upon a time, I was super gung-ho about separating PEcAn across a dozen or so module repos, but I've since come around to the advantages of having all the code be shipped together. At least for now, I think it makes sense to ship `pecanapi` with PEcAn because I definitely foresee API changes that are tied to specific source code changes (case in point, the PFT constants discussion below). If we ever reach such a point of stability that that they can be decoupled, then it could be worth revisiting pulling this out into its own repo.

I agree that it's worth looking at places where other PEcAn packages could use `pecanapi` functionality.

Reply:
> That said, if you did use pecanapi in the main pecan code, that would nix the idea of making it a separate repo

I see I’m well outvoted and will pipe down, but I don’t see how this follows. At some point when it’s not derailing a PR I’d like to hear y’all’s  philosophy on this.